tests/resources/Nimbus-Util.robot

```diff
@@ -451,6 +451,44 @@ Power Off Host
     ${out}=  Execute Command  poweroff -d 0 -f
     Close connection
 
+Static IP Address Create
```

Comment:
I would rename this keyword to something like `Create Simple VC Cluster Testbed With Static IP`

Reply:
Done.